src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -246,6 +246,45 @@ public class foo
 }", "returns");
         }
 
+        [WorkItem(8627, "https://github.com/dotnet/roslyn/issues/8627")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ReadWritePropertyReturns()
+        {
+            await VerifyItemExistsAsync(@"
+public class foo
+{
+    
+    /// $$
+    public void bar { get; set; }
+}", "returns");
+        }
+
+        [WorkItem(8627, "https://github.com/dotnet/roslyn/issues/8627")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ReadOnlyPropertyReturns()
+        {
+            await VerifyItemExistsAsync(@"
+public class foo
+{
+    
+    /// $$
+    public void bar { get; }
+}", "returns");
+        }
+
+        [WorkItem(8627, "https://github.com/dotnet/roslyn/issues/8627")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task WriteOnlyPropertyReturns()
```

Comment:
> Can you make the test name describe the expected result?

Sure.

> You have other tests in this PR that could also use name updates :)

Can you give me an example of what you're looking for? More of the above?